aea/helpers/profiling.py

```diff
@@ -208,18 +219,44 @@ def get_profile_data(self) -> Dict:
         }
 
 
-def get_most_common_objects_in_gc(number: int = 15) -> List[Tuple[str, int]]:
+def get_object_type(obj: Any) -> str:
+    """Returns a string representation of an object's type"""
+    return str(
+        getattr(obj, "__class__", type(obj)).__name__
+    )  # not all objects have the __class__ attribute
```

Comment:
`__name__` is already a string. Remove comment or add it on a separate line above to keep the return a single line and hence more readable. More specifically: built-in types have no `__class__` attribute